Glimpse.ApplicationInsights.Sample/Glimpse.ApplicationInsights.Sample.csproj

```diff
@@ -40,6 +42,24 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <PublishDatabaseSettings>
+      <Objects>
+        <ObjectGroup Name="DefaultConnection-Implementación" Order="1">
+          <Destination Path="Data Source=tcp:nm188qncc0.database.windows.net,1433%3bInitial Catalog=glimpseai_db%3bUser Id=glimpse%40nm188qncc0%3bPassword=Glim%40Ai123_%3b" />
```

Comment:
have you intentionally pushed your credentials here? Read those from web.config and allow to configure from the portal so credentials wouldn't be open for public